doc/sphinx/language/core/basic.rst

```diff
@@ -242,6 +242,7 @@ rest of the |Coq| manual: :term:`terms <term>` and :term:`types
         | @term_generalizing
         | [| {*; @term } %| @term {? : @type } |] {? @univ_annot }
         | @term_ltac
+        | ltac : ( @ltac_expr )
```

Comment:
This is not SSR specific (just in case)

Reply:
Not a problem.  doc_grammar always processes the entire grammar.  Since it's currently not being run for every PR changing the grammar, it picks up a few unrelated changes.

Reply:
There's an issue with this change though, which is that `ltac:(@ltac_expr)` is already what is encompassed into `@term_ltac`, so I don't really understand why this suddenly appears here.